guides/common/modules/con_security-considerations.adoc

```diff
@@ -0,0 +1,55 @@
+[id="security-considerations"]
+= Security considerations in {Project}
+
+When deploying {Project}, consider implementing security features that enhance the overall security of your system.
+{Project} supports multiple security mechanisms to provide additional layers of protection, which you can enable during or after the installation of {Project}, if your use case requires them. 
+
+[NOTE]
+====
+These security considerations apply to both {ProjectServer} and {SmartProxyServer} environments.
+====
+
+
+Federal Information Processing Standards (FIPS) Mode::
+{Project} supports running on FIPS-enabled {RHEL} hosts to comply with security standards for cryptographic modules. 
+However, {Project} itself is not FIPS-certified. In addition, you cannot enable FIPS mode after the installation of {Project}.
++
+FIPS mode enforces the use of validated cryptographic algorithms and modules.
+Enabling FIPS mode ensures that all cryptographic operations within the system adhere to strict security standards.
++
+For more information, see {RHELDocsBaseURL}9/html/security_hardening/switching-rhel-to-fips-mode_security-hardening[Switching RHEL to FIPS mode] in _{RHEL}{nbsp}9 Security hardening_.
++
+[NOTE]
+====
+{Project} supports DEFAULT and FIPS crypto-policies.
+The FUTURE crypto-policy is not supported for {Project} and {SmartProxy} installations.
+The FUTURE policy is a stricter forward-looking security level intended for testing a possible future policy.
+For more information, see {RHELDocsBaseURL}9/html/security_hardening/using-the-system-wide-cryptographic-policies_security-hardening[Using system-wide cryptographic policies] in _{RHEL}{nbsp}9 Security hardening_.
+====
+
+File Access Policy Daemon (`fapolicyd`)::
+By enabling `fapolicyd` on your {ProjectServer}, you can monitor and control file and directory access, which provides an additional security layer.
++
+This feature helps prevent unauthorized code execution and enhances system integrity.
+You can enable or disable `fapolicyd` on both {ProjectServer} and {SmartProxyServer} at any time, depending on your security requirements.
++
+For additional information about `fapolicyd`, including instructions on enabling it, see {RHELDocsBaseURL}9/html/security_hardening/assembly_blocking-and-allowing-applications-using-fapolicyd_security-hardening[Blocking and allowing applications by using fapolicyd] in _{RHEL}{nbsp}9 Security hardening_.
+
+Security-Enhanced Linux (SELinux)::
+{Project} supports SELinux, a mandatory access control system that restricts system access and reduces the risk of security vulnerabilities.
+SELinux enforces policies that define which processes can access specific system resources, providing protection against unauthorized access and exploitation.
++
+Running {Project} with SELinux in enforcing mode is recommended to maintain a high-security environment.
```

Comment:
suggestion on the wording: "is recommended" -> "{Team} recommends running {Project} ..."